src/Executor/ExecutionResult.php

```diff
@@ -142,10 +142,15 @@ public function toArray(int $debug = DebugFlag::NONE) : array
                 return array_map($formatter, $errors);
             };
 
-            $result['errors'] = $errorsHandler(
+            $handledErrors = $errorsHandler(
                 $this->errors,
                 FormattedError::prepareFormatter($this->errorFormatter, $debug)
             );
+
+            // While we know that there were errors initially, they might have been discarded
+            if ($handledErrors !== []) {
+                $result['errors'] = $handledErrors;
+            }
```

Comment:
I realise that this may be a tad late (since it's merged), but the comment suggests this array should be named `$unhandledErrors` for clarity? Since discarded errors are probably handled by an error handler and recovered from. The remaining errors are unrecoverable/unhandled, those are returned to the client.

Reply:
Handling the errors could also mean to translate the messages, format them nicely, whatever!

Reply:
Ah I see :D Yeah then I suppose it makes sense 🙂